app/utils/createEntityReducer.js

```diff
@@ -10,36 +10,36 @@ import type { Reducer, AsyncActionType } from 'app/types';
 type EntityReducerOptions = {
   key: string,
   types: {
-    fetch?: ?AsyncActionType | Array<?AsyncActionType>,
+    fetch?: AsyncActionType | Array<AsyncActionType>,
     mutate?: ?AsyncActionType
   },
   mutate?: Reducer,
   initialState?: Object
 };
 
-export function fetching(fetchType?: ?AsyncActionType) {
+export function fetching(fetchTypes?: Array<AsyncActionType>) {
```

Comment:
Regarding you comment about `fetchType(s)` being optional should the `?` be removed here?

Reply:
Imo. yes.

Without a fetchType, this does nothing

Reply:
My thoughts exactly